pallets/ocex/src/tests.rs

```diff
@@ -2363,65 +2369,69 @@ pub fn test_allowlist_with_limit_reaching_returns_error() {
 }
 
 use crate::{
-	settlement::{add_balance, process_trade, sub_balance},
+	settlement::{add_balance, sub_balance},
 	sr25519::AuthorityId,
 	storage::OffchainState,
 };
 use polkadex_primitives::ingress::{HandleBalance, HandleBalanceLimit};
 
-#[test]
-fn test_set_balances_with_bad_origin() {
-	new_test_ext().execute_with(|| {
-		assert_ok!(OCEX::set_exchange_state(RuntimeOrigin::root(), true));
-		let vec_of_balances: Vec<HandleBalance<AccountId32>> = vec![];
-		let bounded_vec_for_alice: BoundedVec<HandleBalance<AccountId>, HandleBalanceLimit> =
-			BoundedVec::try_from(vec_of_balances).unwrap();
-
-		assert_noop!(OCEX::set_balances(RuntimeOrigin::none(), bounded_vec_for_alice), BadOrigin);
-	});
-}
-
-#[test]
-pub fn test_set_balances_when_exchange_is_not_pause() {
-	new_test_ext().execute_with(|| {
-		assert_ok!(OCEX::set_exchange_state(RuntimeOrigin::root(), true));
-		let vec_of_balances: Vec<HandleBalance<AccountId32>> = vec![];
-		let bounded_vec_for_alice: BoundedVec<HandleBalance<AccountId>, HandleBalanceLimit> =
-			BoundedVec::try_from(vec_of_balances).unwrap();
-
-		assert_noop!(
-			OCEX::set_balances(RuntimeOrigin::root(), bounded_vec_for_alice),
-			Error::<Test>::ExchangeOperational
-		);
-	});
-}
-
-#[test]
-pub fn test_set_balances_when_exchange_is_pause() {
-	let account_id = create_account_id();
-	new_test_ext().execute_with(|| {
-		assert_ok!(OCEX::set_exchange_state(RuntimeOrigin::root(), false));
-		let mut vec_of_balances: Vec<HandleBalance<AccountId32>> = vec![];
-		vec_of_balances.push(HandleBalance {
-			main_account: account_id,
-			asset_id: AssetId::Polkadex,
-			free: 100,
-			reserve: 50,
-		});
-		let bounded_vec_for_alice: BoundedVec<HandleBalance<AccountId>, HandleBalanceLimit> =
-			BoundedVec::try_from(vec_of_balances).unwrap();
-
-		assert_eq!(
-			OCEX::set_balances(RuntimeOrigin::root(), bounded_vec_for_alice.clone()),
-			Ok(())
-		);
-		let blk = frame_system::Pallet::<Test>::current_block_number();
-		assert_eq!(
-			OCEX::ingress_messages(blk)[1],
-			IngressMessages::SetFreeReserveBalanceForAccounts(bounded_vec_for_alice)
-		);
-	});
-}
+//FIXME: This test case is failing. Check if it relevant ot not
```

Comment:
Please do. But I assume it is not, since extrinsic was removed by @Gauthamastro [here](https://github.com/Polkadex-Substrate/Polkadex/pull/884/files#diff-2ff81ab6e292e9586175a812766f54c0f78ce2d05d3eddc851c4da48b6cf3a15L866-L892).